.github/auto-me-bot.yml

```diff
@@ -2,5 +2,6 @@
 ---
 pr:
   conventionalCommits:
+    'body-max-line-length': [2, 'always', 150]
```

Comment:
Maybe this should be under the `rules` directive and not directly under the `conventionalCommits` directive?

That way we can keep some kind of correlation to the actual [configuration object](https://commitlint.js.org/#/reference-configuration?id=javascript), so we can perhaps more easily add more configuration values in the future like `parserPreset`, `formatter`, and etc.

https://github.com/TomerFi/auto-me-bot/issues/45#issuecomment-1200277038